README.md

```diff
@@ -40,6 +40,14 @@ INSTALLED_APPS = (
 
 `django_better_admin_arrayfield.models.fields.ArrayField` is a drop-in replacement for standard Django `ArrayField`.
 
+Import it like below and use it in your model class definition.
+```python
+from django_better_admin_arrayfield.models.fields import ArrayField
+```
+
+Import `DynamicArrayMixin` like below syntax
```

Comment:
please change it to use triple ticks instead of single


Import it like below
```python
from django_better_admin_arrayfield.models.fields import ArrayField
```

Reply:
@gradam I have updated please review and thanks for making this awesome tool, this is really helpful.